pygeos/io.py

```diff
@@ -13,7 +14,7 @@ def to_wkt(
     trim=True,
     output_dimension=3,
     old_3d=False,
-    **kwargs
+    **kwargs,
```

Comment:
This might give syntax errors on py 3.5? (another reason to drop it ;-))

Reply:
I just ran the black code formatter, which makes some guesses about what you want. This will break 3.5 indeed.

Reply:
You have a command line argument to ensure the output is py3.5 compatible: https://github.com/psf/black#command-line-options